packs/victorops/config.yaml

```diff
@@ -0,0 +1,4 @@
+api_key: 20131114/alert/8f048ed9-66af-4406-aa9c-35facef20829
+routing_key: even_team
+debug: false
+url: https://alert.victorops.com/integrations/generic
```

Comment:
Yes, actually this would be changed for every account who wants to use API, URL would be same but api key and routing key needs to be changed. I will add a readme for how to obtain these api key and routing key.